mangum/backends/base.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class WebSocketBackend:  # pragma: no cover
```

Comment:
We should add `metaclass=ABCMeta` in arguments if it is accepted in `@dataclass`.

Reply:
I'm sorry, mypy don't accept ABCMeta in `@dataclass`

Reply:
I'm sorry, mypy doesn't accept `ABCMeta` with `@dataclass`.